infra.bs

```diff
@@ -547,6 +547,27 @@ ordered sets; implementations can optimize based on the fact that the order is n
 <a for=list>prepend</a> operation otherwise.
 
 
+<h4 id=tuples>Tuples</h4>
+
+<p>Some lists are designated as <dfn export lt=tuple>tuples</dfn>. A tuple is a <a>list</a> with the
+additional semantic that its <a for=list>size</a> must not change and that its
+<a for=tuple>items</a> can be indexed by <dfn export for=tuple>name</dfn>.
+
+<p>For notational convenience, a literal syntax can be used to express <a>tuples</a>, by surrounding
+the tuple with ( ) characters and separating the <a for=tuple>items</a> by comma. The
+<a for=tuple>items</a> can be preceded by <a for=tuple>name</a> and : character if the
+<a for=tuple>names</a> are not clear from context. An indexing syntax can be used by providing a
+zero-based index and named index into a list inside square brackets.
```

Comment:
Yeah, I guess you're right.

Your pair example is also interesting. I wonder if we should have dedicated support for those. There's numerous places that use the / syntax that way.